language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionModuleId.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.codeaction;
+
+import io.ballerina.compiler.api.ModuleID;
+import io.ballerina.compiler.syntax.tree.ImportDeclarationNode;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Represents a module information in ballerina.
+ * <p>
+ * Allows convenient transformation of ImportDeclarationNode node model representation for org-name, module-name,
+ * version and alias.
+ *
+ * @since 2.0.0
+ */
+public class CodeActionModuleId implements ModuleID {
+    private static final String ORG_SEPARATOR = "/";
+    private final String orgName;
+    private final String moduleName;
+    private final String version;
+    private final String alias;
```

Comment:
Added alias to support ModuleID interface. I think we need to remove `alias` from this class since it doesn't affect the equality of the two separate imports. `equal` implementation ignores it here.